specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,48 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the key, from your Azure KeyVault, you wish to use to protect your data. "
+        },
+        "keyVaultKeyVersion": {
+          "type": "string",
+          "description": "The specific version of the key, from your Azure KeyVault, you wish to use to protect your data."
```

Comment:
I forget -- is "key version" a Key Vault concept, or something we invented? If it's the latter, it will require more explanation. If it's the former, I recommend using a similar structure to what I proposed above: "The version of a key that you create in Azure KeyVault that will be used to encrypt your data at rest."

Reply:
Also, should it be "KeyVault" or "Key Vault"? We should be consistent with other documentation in how we spell this.

Reply:
The version is a 32-byte alphanumeric string, like our api-keys.   It's not exactly what I expected to see for a "version" value.

Reply:
@brjohnstmsft , key version is a key vault concept
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.keyvault.objectidentifier.version?view=azure-dotnet#Microsoft_Azure_KeyVault_ObjectIdentifier_Version